src/nxt_otel.h

```diff
@@ -11,13 +11,23 @@
 
 // forward declared
 struct nxt_http_field_t;
+struct nxt_conf_validation_t;
+struct nxt_conf_value_t;
 
-extern void   nxt_otel_send_trace(void *);
-extern void * nxt_otel_get_or_create_trace(u_char *);
-extern void   nxt_otel_init(void (*)(u_char*));
-extern void   nxt_otel_copy_traceparent(u_char *, void *);
-extern void   nxt_otel_add_event_to_trace(void *, u_char *, u_char *);
-extern void   nxt_otel_end_span(void *);
+extern void     nxt_otel_send_trace(void *);
+extern void   * nxt_otel_get_or_create_trace(u_char *);
+extern void     nxt_otel_init(void (*)(u_char*), const char *, const char *, double);
+extern void     nxt_otel_copy_traceparent(u_char *, void *);
+extern void     nxt_otel_add_event_to_trace(void *, u_char *, u_char *);
+extern uint8_t  nxt_otel_is_init();
+extern void     nxt_otel_uninit();
```

Comment:
Currently functions without a prototype (i.e. no arguments specified) means a function that takes an unspecified number of arguments (I believe this will change in C23 to make it match C++ semantics), so assuming these functions really don't take any parameters, please make them (void).